distribution/zip/jballerina/bin/ballerina

```diff
@@ -159,13 +159,16 @@ done
 if [ -n "${BAL_JAVA_DEBUG+set}" ]; then
     if [ -z "$BAL_JAVA_DEBUG" ]; then
         echo "Please specify the debug port for the BAL_JAVA_DEBUG variable"
-	exit 1
+        exit 1
     else
-	if [ -n "$JAVA_OPTS" ]; then
-	  echo "Warning !!!. User specified JAVA_OPTS may interfere with BAL_JAVA_DEBUG"
-	fi
-	# 'quiet=y' avoids breaking LSP protocol in debug mode
-	JAVA_OPTS="$JAVA_OPTS -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=$BAL_JAVA_DEBUG,quiet=y"
+      if [ -n "$JAVA_OPTS" ]; then
+        echo "Warning !!!. User specified JAVA_OPTS may interfere with BAL_JAVA_DEBUG"
+      fi
+      if [ -n "${DEBUG_OPTS+set}" ]; then
```

Comment:
How about being more specific by using something like `BAL_DEBUG_OPTS` as the env var name instead, just to avoid confusions when working in environments which have multiple debugger related entities? :)

Reply:
+1 for the suggestion

Reply:
+1 that makes sense to me. Will update the PRs